recipes/cosolvkit/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "cosolvkit" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cosolvkit-{{ version }}.tar.gz
+  sha256: 960aa8bf95db081c62df5bb3634d11e06b4a3f28740b1400de05b7c62503d44b
+
+build:
+  skip: True  # [win or aarch64]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python ==3.10
```

Comment:
There is no metadata on the dependencies but the classifiers imply that it can be noarch and would work with py38 to py312.

Reply:
Please address all the review comments. Before pinging the team again. This package seems to be fine for noarch.

Reply:
Hi, thanks for the comment. This package is not fine for noarch since one of its dependencies doesn't work with Windows. Furthermore, other dependencies work fine just with Python3.10 for the moment, hence I need to pin the Python verison.

What could I do differently? I'm sorry if I'm asking very basic stuff, but this is my first time creating a conda-forge recipe.

Reply:
@ocefpaf ping

Reply:
@conda-forge-admin, please asking for support or review.